src/sentry/api/helpers/group_index/update.py

```diff
@@ -409,18 +414,9 @@ def update_groups(
                             release_version=release.version,
                         )
 
-                        if (
-                            features.has(
-                                "organizations:releases-resolve-next-release-semver-fix",
-                                project.organization,
-                            )
-                            and follows_semver
-                        ):
-                            current_release_version = get_release_to_resolve_by(projects[0]).version
-                        else:
-                            current_release_version = get_current_release_version_of_group(
-                                group=group, follows_semver=follows_semver
-                            )
+                        current_release_version = get_current_release_version_of_group(
```

Comment:
Since the fix is now within `get_current_release_version_of_group` we do not need the if/else block.